### PR TITLE
chore: remove user feature flag

### PR DESCRIPTION
**Pull request by nickoferrall**
Fix https://github.com/ParabolInc/parabol/issues/7764

If the PR is approved as it is, I'll update our Notion docs to reflect the change.

### To test

- [ ] Run the `updateFeatureFlag` mutation with `addFlag` as true. Go to pgadmin and see that a flag was added
- [ ] Run the `updateFeatureFlag` mutation with `addFlag` as false. Go to pgadmin and see that the flag was removed

### CODE REVIEW

File: packages/client/mutations/UpdateFeatureFlagMutation.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import graphql from 'babel-plugin-relay/macro'
+import {commitMutation} from 'react-relay'
+import {StandardMutation} from '../types/relayMutations'
+import {UpdateFeatureFlagMutation as TUpdateFeatureFlagMutation} from '../__generated__/UpdateFeatureFlagMutation.graphql'
+
+graphql`
+  fragment UpdateFeatureFlagMutation_notification on UpdateFeatureFlagPayload {
+    user {
+      id
+      # add flag here
```
Comment (igorlesnenko) [inline packages/client/mutations/UpdateFeatureFlagMutation.ts:10]: I'm not sure what that comment means? Do we use this mutation on client? Maybe remove it completely, or do we plan to use it in future?

**Review by igorlesnenko — APPROVED** [review]
Works!

File: packages/client/mutations/UpdateFeatureFlagMutation.ts (hunk 1 above)
Comment (nickoferrall) [reply packages/client/mutations/UpdateFeatureFlagMutation.ts:10]: I took this from the `AddFeatureFlagMutation` - I think if we want to notify users, we can include the flag here

**Comment by nickoferrall** [comment]
@igorlesnenko I meant to include the `One Review Required` label in this PR. Any objections to me merging this PR or do you think I should request a Maintainer Review?

**Comment by igorlesnenko** [comment]
@nickoferrall No objections

File: packages/client/mutations/UpdateFeatureFlagMutation.ts (hunk 1 above)
Comment (igorlesnenko) [reply packages/client/mutations/UpdateFeatureFlagMutation.ts:10]: Yeah, I mean, I don't see we use `AddFeatureFlagMutation` or `UpdateFeatureFlagMutation` on client. We export it but don't import

Comment (nickoferrall) [reply packages/client/mutations/UpdateFeatureFlagMutation.ts:10]: I think we used to let users add themselves to a feature using a cheat code, like if they typed in "Spotlight", they'd get access to it.

I could see that happening again in the future. For instance, if they could request access to the Insights beta feature and we'd automatically turn it on. So I think it's safe to keep it

**Pull request merged**